source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -126,19 +133,20 @@ namespace quic {
 
 using QuicLogLevel = spdlog::level::level_enum;
 
-static const QuicLogLevel TRACE = spdlog::level::trace;
-static const QuicLogLevel DEBUG = spdlog::level::debug;
-static const QuicLogLevel INFO = spdlog::level::info;
-static const QuicLogLevel WARNING = spdlog::level::warn;
-static const QuicLogLevel ERROR = spdlog::level::err;
-static const QuicLogLevel FATAL = spdlog::level::critical;
-
+enum {
```

Comment:
At first glance I did not see where these are used. Now I see they are used in macros via `quic::LogLevel##severity`.

Suggest instead using `enum class LogLevel` and then you can use `quic::LogLevel::severity` in the macros, and use values of TRACE, DEBUG, etc.

Also add a comment here saying that these have to be named that way to match usages in code imported into Envoy, which is why this is not conforming exactly to Envoy Style guide, which would name enum values a little differently.

Reply:
Thanks for the review! `quic::LogLevel::severity` wouldn't work because the preprocessor would then replace `DEBUG` with `1` on some Envoy macOS builds which are built with `-DDEBUG=1`. I've added a comment that should clarify this point, and also why these exist, and why their formatting is non-standard.